pkgs/games/openra/builds.nix

```diff
@@ -0,0 +1,34 @@
+pkgs:
+
+let
+  f = import ./common.nix;
+  common = f (builtins.intersectAttrs (builtins.functionArgs f) pkgs // {
```

Comment:
Can't you just use `pkgs.callPackage`?

Reply:
Nope, I tried, but `callPackage` [makes it overridable](https://github.com/NixOS/nixpkgs/blob/c6e043d57c2cc69a1068a5e6e34b385942876d0e/lib/customisation.nix#L112), adding additional attributes to the returned attribute set, causing the argument destructering of `mod.nix` and `engine.nix` to fail due to the additional attribute names.